products/BellHybrid/services/db/include/db/SystemSettings.hpp

```diff
@@ -35,4 +35,8 @@ namespace bell::settings
         constexpr inline auto tone     = "bedtime_tone";
         constexpr inline auto duration = "bedtime_duration";
     } // namespace Bedtime
+    namespace Layout
+    {
+        constexpr inline auto defaultLayout = "layout";
```

Comment:
what is this var? I guess it is just the current layout. Why word "default"? Maybe it would be more clear without it?